### ISSUE TITLE
Support working with stdin and stdout

### ISSUE THREAD

**Issue by radomirbosak** [chamkank/hone#1]
When using the CLI sometimes it's useful to not use files, but stdin/stdout instead. If allows further processing with e.g the [jq](https://github.com/stedolan/jq) tool.

It would be nice to be able to do this:
```console
$ echo 'name,age (years),weight (kg),birth day,birth month,birth year,adopted,adopted_since
Tommy,5,3.6,11,April,2011,TRUE,2012' | hone > output.json
```

```console
$ hone input.csv
[
  {
    "adopted": "TRUE",
    "adopted_since": "2012",
    "age (years)": "5",
    "birth": {
      "day": "11",
      "month": "April",
      "year": "2011"
    },
    "name": "Tommy",
    "weight (kg)": "3.6"
  }
]

$ hone input.csv | jq '.[0]["name"]'
"Tommy"
```

**Comment by chamkank** [comment]
Sorry for the extremely late response, I think this a great idea. It's been implemented in #12, but not exactly as you specified. In order to specify to use stdin/stdout you'll have to use `-` as the filename. 

Example:
```bash
$ cat examples/example_a.csv | hone - -
[{'adopted_since': '2012', 'adopted': 'TRUE', 'birth': {'year': '2011', 'month': 'April', 'day': '11'}, 'weight (kg)': '3.6', 'age (years)': '5', 'name': 'Tommy'}, {'adopted_since': 'N/A', 'adopted': 'FALSE', 'birth': {'year': '2015', 'month': 'May', 'day': '6'}, 'weight (kg)': '8.2', 'age (years)': '2', 'name': 'Clara'}, {'adopted_since': '2017', 'adopted': 'TRUE', 'birth': {'year': '2011', 'month': 'August', 'day': '21'}, 'weight (kg)': '3.3', 'age (years)': '6', 'name': 'Catnip'}, {'adopted_since': '2018', 'adopted': 'TRUE', 'birth': {'year': '2015', 'month': 'January', 'day': '18'}, 'weight (kg)': '3.1', 'age (years)': '3', 'name': 'Ciel'}]
```

This feature has been released with [v0.1.5](https://pypi.org/project/hone/0.1.5/).

**Comment by radomirbosak** [comment]
Nice, thank you!
